### PR TITLE
Enable early stopping for GJK solver when computing contacts

### PR DESCRIPTION
**Pull request by jcarpent**
With minor cosmetic change

### CODE REVIEW

File: include/hpp/fcl/collision_data.h [hunk 1]
```diff
@@ -206,6 +207,9 @@ struct HPP_FCL_DLLAPI CollisionRequest : QueryRequest
 
   /// Whether a lower bound on distance is returned when objects are disjoint
   bool enable_distance_lower_bound;
+  
+  /// Whether GJK stops when it proves that the distance between two geometries is greater than distance_upper_bound.
+  bool enable_early_stopping;
```
Comment (jmirabel) [inline include/hpp/fcl/collision_data.h:212]: I think it is sufficient to set the `distance_upper_bound` to infinity (or max, as in the constructor).

Comment (jcarpent) [reply include/hpp/fcl/collision_data.h:212]: OK. Having the double mechanism seems to be safer at some point. What do you think?

**Review by jmirabel — COMMENTED** [review]
There are plenty of syntax changes. At the moment, I am not aware of any syntax rules so I would prefer not to include them. You changed some stuff to the way you like and there is nothing that prevents someone to change it later to the way he/she likes.

If we want to do it, we can do it:
- in a separate PR,
- done on all the code,
- by some syntax checker like clang-format.

**Comment by jmirabel** [comment]
And thanks for this change.

File: include/hpp/fcl/collision_data.h (hunk 1 above)
Comment (jmirabel) [reply include/hpp/fcl/collision_data.h:212]: Having a double mechanism is double-folded. You also need to make sure you updated both.

You can add functions to deal with `distance_upper_bound` if you want this to be *safe* but I don't think it is necessary.

**Comment by florent-lamiraux** [comment]
There are still many modifications of indentation or line breaking that pollute the history. Can you please remove them ?

**Comment by jcarpent** [comment]
Done

**Comment by florent-lamiraux** [comment]
Almost.

**Pull request merged**